integration-tests/benchmark/keeper_test.go

```diff
@@ -98,7 +98,7 @@ LimitDefault = 5_000_000`
 			},
 		},
 		"stateful": true,
-		"capacity": "20Gi",
+		"capacity": "1Gi",
```

Comment:
Fine for now. Some time back, had run into some issues with DB capacity during high throughput tests on fast chains for long duration. Will increase if needed.

Reply:
If we need to we can make this configurable from the github workflow inputs.